### DISCUSSION TITLE
Add the ability to delete the system message about joining a group

### DISCUSSION THREAD

**Discussion by fear-rus** [umputun/tg-spam#91] [Ideas]
I suggest adding the ability to delete the system message about joining the group after deleting the message and banning the user. When there is a lot of spam, and the bot deletes it, then there are system messages about joining, which look like spam pages in the group. Is it possible to automatically delete messages about joining a group of spambots that have been deleted?

**Comment by umputun** [comment]
I never tried it, but it seems possible with TG bot API.

Theoretically, it needs to check `update.Message.NewChatMembers` and record this info associated with the user (probably a part of Locator). To remove it, a regular removal for this message ID should work. As you can see, this seems to be doable but not a trivial change. I'll make a ticket from this discussion and label it "help needed", maybe someone will take care of this.
